setup.py

```diff
@@ -72,6 +72,7 @@
         omp_cxxflags.insert(0, '-Xpreprocessor')
         libs = ['omp']
         ldflags = ['-Wl,-rpath,/usr/local/lib']
```

Comment:
```suggestion
        ldflags = ['-Wl,-rpath,/usr/local/lib', '-Wl,-rpath,/opt/homebrew/lib']
```

This is what you tried?